UnityEngine.TestRunner/NUnitExtensions/UnityTestAssemblyBuilder.cs

```diff
@@ -65,6 +65,8 @@ public IEnumerator<ITest> BuildAsync(Assembly[] assemblies, TestPlatform[] testP
         {
             var productName = string.Join("_", m_ProductName.Split(Path.GetInvalidFileNameChars()));
             var suite = new TestSuite(productName);
+
+            var lastYieldTime = Time.realtimeSinceStartup;

```

Comment:
I'd generally prefer using Stopwatch, but I guess this works just as well and is not affected by time scale.